Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -1608,7 +1610,7 @@ int CLuaEngineDefs::EngineGetModelPhysicalPropertiesGroup(lua_State* luaVM)
 
     if (!argStream.HasErrors())
     {
-        if (iModelID < 0 || iModelID >= 20000)
+        if (iModelID < 0 || iModelID >= GetBaseIDforTXD())
         {
             argStream.SetCustomError("Expected model ID in range [0-19999] at argument 1");
```

Comment:
This error message still uses a hardcoded number. I think the intent here is to derive the number from the base TXD ID.

```suggestion
        if (iModelID < 0 || iModelID >= GetBaseIDforTXD())
        {
            argStream.SetCustomError(SString("Expected model ID in range [0-%d] at argument 1", GetBaseIDforTXD() - 1));
```

Reply:
Right. I'll alter it.

Reply:
Done. Now it's all complete. It's all made to work.